java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -547,6 +548,26 @@ public void hdel_success() {
         assertEquals(value, payload);
     }
 
+    @SneakyThrows
+    @Test
+    public void select_returns_success() {
+        // setup
+        CompletableFuture<String> testResponse = mock(CompletableFuture.class);
+        Long index = 55L;
```

Comment:
even though you're using a mock -lets fix the tests to index that is supported by redis (0-15)